backend/views/syson-tree-explorer-view/src/main/java/org/eclipse/sirius/web/application/views/explorer/services/api/IExplorerServices.java

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
```

Comment:
This class should be removed once https://github.com/eclipse-sirius/sirius-web/pull/4256 is merged